codelabs/keyboard_navigation/keyboard_navigation.md

```diff
@@ -46,59 +46,25 @@ You will use the (`npx @blockly/create-package app`)[https://www.npmjs.com/packa
 
 ## Terminology
 
-A [**Marker**](https://developers.google.com/blockly/reference/js/blockly.marker_class) holds a location and is not movable.
+A [**Marker**](https://developers.google.com/blockly/reference/js/blockly.marker_class) holds a location and is not movable.  A marker can be used to mark a location on the workspace that you can use later, such as marking the spot where you'll drop a block from the toolbox.
 
 A [**Cursor**](https://developers.google.com/blockly/reference/js/blockly.cursor_class) is a marker that can move. It extends a `Blockly.Marker` but adds logic to allow the marker to move through the blocks, inputs, fields, connections and workspace coordinates.
 
 The below image displays different parts of a block that a user can navigate to using keyboard navigation.
 
 ![Displays the different parts of a block. The previous connection on the top of a block. The next connection on the bottom of a block. Input value as a cut out of a puzzle piece. The statement input as a connection inside of a block. The output connection as a puzzle piece.](./block_terms.png)
 
-
-## Define and set a custom cursor
-
-To start, create a new directory at `src/cursors` and add a file inside named `custom.js`.
-
-At the top of the file, you first need to import `blockly/core`:
-
-```js
-import * as Blockly from 'blockly/core';
-```
-
-Then define your new custom cursor and have it extend the base cursor, `Blockly.Cursor`:
-
-```js
-export class CustomCursor extends Blockly.Cursor {
-  constructor() {
-    super();
-  }
-}
-```
-
-Import your cursor in `src/index.js`.
-
-```js
-import {CustomCursor} from './cursors/custom';
-```
-
-After injecting the workspace, use its `MarkerManager` to set your new custom cursor:
-
-```js
-// Add CustomCursor to workspace
-ws.getMarkerManager().setCursor(new CustomCursor());
-```
-
 ## Initialize NavigationController plugin
 
-Now you must initialize `NavigationController` in `index.js`.  `NavigationController` is the class in charge of registering all keyboard shortcuts.
+First, you will want to initialize a `NavigationController` in `index.js`.  `NavigationController` is the class in charge of registering all keyboard shortcuts.
 
-First, import `NavigationController` at the top of `index.js`:
+Import `NavigationController` at the top of `index.js`:
 
 ```js
 import {NavigationController} from '@blockly/keyboard-navigation';
 ```
 
-Then, after injecting the workspace, instantiate an instance of `NavigationController`, initialize it, and add it to our workspace:
+Then, somewhere after the existing code that injects the workspace, instantiate an instance of `NavigationController`, initialize it, and add it to your workspace:
```

Comment:
Since we're spiffing it up, let's fix the redundant/repetitive wording:
```suggestion
Then, somewhere after the existing code that injects the workspace, create an instance of `NavigationController`, initialize it, and add it to your workspace:
```
Also: possibly "the workspace"?